src/projects/TodoList/index.js

```diff
@@ -69,7 +97,13 @@ class TodoList extends React.Component {
             <p className='pr-1'><strong>Add a new item:</strong></p>
           </div>
           <div>
-            <input className='form-control' value={this.state.newItem} placeholder='enter text here...' onKeyPress={this.pressEnter} onChange={e => this.setState({ newItem: e.target.value })}/>
+            <input
+              className='form-control'
```

Comment:
If you want to do multiple classNames, you just add them into the string
```jsx
<input className="some-class another-class a-third-class" />
```

Reply:
cool thanks